### PR TITLE
chore: updating scripts and deploying token-based service contracts on chiado

### PR DESCRIPTION
**Pull request by kupermind**
- Deploying OperatorWhitelist, ServiceRegistryTokenUtility and Service ManagerToken on chiado;
- Verifying the ServiceManagerToken setup using the ServiceRegistryL2 contract.

### CODE REVIEW

File: hardhat.config.js [hunk 1]
```diff
@@ -77,8 +77,8 @@ module.exports = {
                 network: "chiado",
                 chainId: 10200,
                 urls: {
-                    apiURL: "https://blockscout.com/gnosis/chiado/api",
-                    browserURL: "https://blockscout.com/gnosis/chiado",
+                    apiURL: "https://gnosis-chiado.blockscout.com/api",
+                    browserURL: "https://gnosis-chiado.blockscout.com/",
```
Comment (kupermind) [inline hardhat.config.js:81]: These are new URLs for the chiado network to correctly do the contract verification.

File: scripts/deployment/l2/bridges/gnosis/test_service_registry_l2_change_manager.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
+/*global process*/
+
+const { ethers } = require("ethers");
+
+async function main() {
```
Comment (kupermind) [inline scripts/deployment/l2/bridges/gnosis/test_service_registry_l2_change_manager.js:5]: Script to change the manager for the ServiceRegistryL2 to swap ServiceManager with ServiceManagerToken on chiado.

File: scripts/deployment/l2/bridges/gnosis/test_service_registry_token_utility_change_drainer.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
+/*global process*/
+
+const { ethers } = require("ethers");
+
+async function main() {
```
Comment (kupermind) [inline scripts/deployment/l2/bridges/gnosis/test_service_registry_token_utility_change_drainer.js:5]: Script fo change the drainer on chiado

File: scripts/proposals/proposal_04_service_registry_token_utility_change_drainer_gnosis.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
+/*global process*/
+
+const { ethers } = require("ethers");
+
+async function main() {
```
Comment (kupermind) [inline scripts/proposals/proposal_04_service_registry_token_utility_change_drainer_gnosis.js:5]: A proposal to change the drainer for the ServiceRegistryTokenUtility on gnosis mainnet (not needed if we do it via the initial deployment scripts)

File: scripts/proposals/proposal_04_service_registry_l2_change_manager_gnosis.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
+/*global process*/
+
+const { ethers } = require("ethers");
+
+async function main() {
```
Comment (kupermind) [inline scripts/proposals/proposal_04_service_registry_l2_change_manager_gnosis.js:5]: A proposal script to change the manager for the ServiceRegistryL2 to swap ServiceManager with ServiceManagerToken on gnosis.

**Review by 77ph — APPROVED** [review]

File: scripts/proposals/proposal_04_service_registry_l2_change_manager_gnosis.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+    console.log("AMBContractProxyHomeAddress to call homeMediator's processMessageFromForeign function with the data:", data);
+
+    const requestGasLimit = "2000000";
+    const timelockPayload = await AMBProxy.interface.encodeFunctionData("requireToPassMessage", [homeMediatorAddress,
+        mediatorPayload, requestGasLimit]);
+
+    const targets = [AMBProxyAddress];
+    const values = [0];
+    const callDatas = [timelockPayload];
+    const description = "Change Drainer in ServiceRegistryL2 on gnosis";
```
Comment (mariapiamo) [inline scripts/proposals/proposal_04_service_registry_l2_change_manager_gnosis.js:75]:
```suggestion
    const description = "Change Manager in ServiceRegistryL2 on gnosis";
```

**Review by mariapiamo — APPROVED** [review]

**Pull request merged**